feconf.py

```diff
@@ -155,6 +155,8 @@ def get_empty_ratings():
 CAN_SEND_EMAILS_TO_USERS = False
 # Whether to send email updates to a user who has not specified a preference.
 DEFAULT_EMAIL_UPDATES_PREFERENCE = False
+# Whther to send membership email when user who has not specified a preference.
```

Comment:
Whether to send an invitation email when the user is granted new role permissions in an exploration.